ripple/src/main/webapp/accountsettings.html

```diff
@@ -16,16 +16,33 @@
     <script src="https://www.gstatic.com/firebasejs/7.15.5/firebase-auth.js"></script>
     <script src="https://www.gstatic.com/firebasejs/7.15.5/firebase-firestore.js"></script>
     <script>
-      firebase.initializeApp(config);
+      var apiKey = config.API_KEY;
```

Comment:
Will this config setup be moved to the config file and called in one line for shorter code?

Reply:
Fixed